include/deal.II/grid/tria.h

```diff
@@ -3073,16 +3073,16 @@ class Triangulation : public Subscriptor
 
   /**
    * Return the total number of used faces, active or not.  In 2D, the result
-   * equals n_lines(), while in 3D it equals n_quads(). Since there are no
-   * face objects in 1d, the function returns zero in 1d.
+   * equals n_lines(), in 3D it equals n_quads(), while in 1D it equals
+   * the number of vertices.
```

Comment:
say "used vertices" to be consistent with the `get_vertices()` and `get_used_vertices()` functions

Reply:
mmm...  this is not the number of used vertices, though. It is the total number of vertices (i.e., all faces, including inactive faces, which, in 1D, are inactive vertices).

Reply:
No, that's not quite right. All used vertices are active, and so `n_face()` should return the same as `n_active_faces()`. But what the `get_vertices()` function returns is an array of point locations of which only some are actually used. The difference between used and unused is the difference between `cell_iterator` and `raw_cell_iterator`.

The point I'm trying to make is that of course the returned number of faces in 1d is equal to the number of vertices *that exists as part of the (logical) mesh*, but it is a subset of the vertices *stored in the data structure `class Triangulation`*.